### PR TITLE
Divyanshu | Test | Provide bulk export support on credit/debit note and purchase bill

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cvdx95y

### CODE REVIEW

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 1]
```diff
@@ -741,3 +749,20 @@ <h5>
...
+<ng-template #bulkExport>
+    <invoice-bulk-export
+        [type]="'receipt'"
+        [dateRange]="{ from: fromDate, to: toDate}"
+        [advanceSearch]="lastListingFilters"
+        [voucherUniqueNames]="selectedReceipts"
+        [totalItems]="
+            selectedReceipts && selectedReceipts.length > 0
+                ? selectedReceipts.length
+                : allReceipts && pageConfiguration.totalItems
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html]:
```suggestion
                : allReceipts && allReceipts.length
```

File: apps/web-giddh/src/app/reports/components/payment-report/payment-report.component.html [hunk 2]
```diff
@@ -594,3 +604,20 @@ <h5>
...
+<ng-template #bulkExport>
+    <invoice-bulk-export
+        [type]="'payment'"
+        [dateRange]="{ from: fromDate, to: toDate}"
+        [advanceSearch]="lastListingFilters"
+        [voucherUniqueNames]="selectedPayments"
+        [totalItems]="
+            selectedPayments && selectedPayments.length > 0
+                ? selectedPayments.length
+                : allPayments && pageConfiguration.totalItems
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/payment-report/payment-report.component.html]:
```suggestion
                : allPayments && allPayments.length
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html]: Done

File: apps/web-giddh/src/app/reports/components/payment-report/payment-report.component.html (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/reports/components/payment-report/payment-report.component.html]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**